docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,90 @@
+# Release 405 (23 Nov 2022)
+
+```{warning}
+Where's release 404? Not found.
+```
+
+## General
+
+* Improve performance of queries that compare `date` columns with
+  `timestamp with time zone` literals. ({issue}`5798`)
+* Improve performance and resource utilization of `INSERT` queries into
+  partitioned and bucketed tables. ({issue}`14718`)
+
+## Security
+
+* Add support for string replacement as part of
+  [impersonation rules](system-file-auth-impersonation-rules). ({issue}`14962`)
+* Fix some `system.metadata` tables improperly showing the names of catalogs
+  which the user cannot access. ({issue}`14000`)
+* Fix `USE` statement improperly showing the names of catalogs and schemas which
+  the user cannot access. ({issue}`14208`)
+
+## BigQuery connector
+
+* Add support for retrying when creating a BigQuery read session. ({issue}`15013`)
+* Fix queries incorrectly executing with the `project-id` specified in the
+  credentials instead of the project ID specified in the `bigquery.project-id`
+  catalog property. ({issue}`14083`)
+
+## Delta Lake connector
+
+* Improve performance and reduce storage requirements when running the `vacuum`
+  procedure on S3-compatible storage. ({issue}`15072`)
+* Improve performance of reading Parquet files for `boolean`, `tinyint`,
+  `short`, `int`, `long`, `float`, `double`, `short decimal`, `UUID`, `time`,
+  `decimal`, `varchar`, and `char` data types. The
+  `parquet.optimized-reader.enabled` catalog configuration property can be set
+  to `false` to disable the optimized implementation. ({issue}`14423`, {issue}`14667`)
+
+## Google Sheets connector
+
+* Rename the `credentials-path` configuration property to
+  `gsheets.credentials-path`, `metadata-sheet-id` to
+  `gsheets.metadata-sheet-id`, `sheets-data-max-cache-size` to
+  `gsheets.max-data-cache-size`, and `sheets-data-expire-after-write` to
+  `gsheets.data-cache-ttl`. ({issue}`15042`)
+
+## Hive connector
+
+* Improve performance of reading Parquet files for `boolean`, `tinyint`,
+  `short`, `int`, `long`, `float`, `double`, `short decimal`, `UUID`, `time`,
+  `decimal`, `varchar`, and `char` data types. The
+  `parquet.optimized-reader.enabled` catalog configuration property can be set
+  to `false` to disable the optimized implementation. ({issue}`14423`, {issue}`14667`)
+* Only allow creating ACID tables with the Thrift metastore. ({issue}`14673`)
```

Comment:
This is a bit misleading - people can think it means you can create "only ACID" tables with Thrift metastore.

Maybe word it as a negative - `Disallow creating ACID tables when not using the Thrift metastore`?